.eslintrc.js

```diff
@@ -0,0 +1,62 @@
+module.exports = {
```

Comment:
I think this file is basically the same as what's in the View repo, so I think we can move this to the devtools repo instead.